### PR TITLE
fix(type-layout): set xs and sm font sizes to caption-01

### LINKED ISSUE #27: TypeLayout Component (with small & x-small) are wrong font-size
## Bug - TypeLayout Component (with small & x-small) are wrong font-size.

**Expected behavior -**

The font-size should follow the label style from Carbon, which is 12px.

**Actual behavior -**

The current font-size follows the body style, which is 14px.

### Steps for reproducing

1.  View component in storybook: https://ibm-security.carbondesignsystem.com/?path=/story/components-typelayout--with-x-small

### PR DESCRIPTION
**Pull request by jendowns**
## Affected issues

- Resolves #27 

## Proposed changes

- set xs and sm font sizes to `caption-01` (12px)

### CODE REVIEW

File: src/components/TypeLayout/_mixins.scss [hunk 1]
```diff
@@ -62,11 +62,13 @@
     }
 
     &--sm {
+      @include carbon--type-style($name: caption-01);
```
Comment (SimonFinney) [inline src/components/TypeLayout/_mixins.scss]: It looks like there's some duplication here - We can reduce that by grouping the selectors

Comment (jendowns) [reply src/components/TypeLayout/_mixins.scss]: 👍 Done

**Pull request merged**